README.md

```diff
@@ -133,6 +133,29 @@ K9s is available on Linux, macOS and Windows platforms.
 ## Building From Source
 
  K9s is currently using go v1.14 or above. In order to build K9s from source you must:
+ installing go language
+
+ 
+ ```bash
+wget https://golang.org/dl/goX.Y.Z.linux-amd64.tar.gz
+
+sudo tar -xvf go1.21.0.linux-amd64.tar.gz -C /usr/local
+
+ ```
+Update your environment variables
+```bash
+export GOROOT=/usr/local/go
```

Comment:
Don't think these are required anymore. PATH should be the only thing one must set. 
Also if we have to include how to set shell envs then installing k9s from source might not be the best option vs downloading the binary directly (Bonus: don't need to install GO!).

Reply:
it's true
I removed it